strawberry/private.py

```diff
@@ -0,0 +1,30 @@
+class Private:
+    """Represent a private field that won't be converted into a GraphQL field
+
+    Example:
+
+    ```
+    import strawberry
```

Comment:
Other docstring examples in the repo use doctest style. May want to stay consistent?

Reply:
Good point. Updated